cmd/nephe-controller/options.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"github.com/spf13/pflag"
+	"gopkg.in/yaml.v2"
+	"os"
+)
+
+const (
+	namePrefix = "nephe-"
```

Comment:
nit: defaultResourcePrefix?

Try to be consistent on this option var name across all files.

Reply:
Solved